src/index.js

```diff
@@ -0,0 +1,119 @@
+let currentTemp = 0;
+
+const updateTemperature = currentTemp => {
+    const tempContainer = document.getElementById("tempNum");
+    tempContainer.textContent = currentTemp;
+    changeTempColor(currentTemp);
+    changeLandscape(currentTemp);
+};
+
+const increaseTemperature = () => {
+    currentTemp += 1;
+    updateTemperature(currentTemp);
+};
+
+const decreaseTemperature = () => {
+    currentTemp -= 1;
+    updateTemperature(currentTemp);
+};
+//garden object
+const changeTempColor = currentTemp => {
+    const tempContainer = document.getElementById("garden-section");
+    let color = "green";
+    if (currentTemp >= 10) {
```

Comment:
Consider how similar all the branches are here. We look for whether the temperature is within a range, and pick a color accordingly. What if we had a list of records that we could iterate through to find the values. Maybe something like
```js
  const tempRecords = [
    { lowerBound: 10, color: "red"},
    { lowerBound: 7, color: "other color"},
    ...
  ];
```

We could simply iterate through the list and find the first record that has a lower bound _lower_ than our temperature as the source of picking the color!

We could take a similar approach for the landscape below, and use a direct dictionary lookup for the sky values (since there's no range required).